gatsby/src/types.d.ts

```diff
@@ -13,3 +13,11 @@ export interface Topic extends PartialTopic {
 	/** Intro sentence (or two) describing the topic */
 	topicSummary: string;
 }
+
+export interface Speciality {
```

Comment:
Can you split this into a partial and a full to mimic the graph ql fragments please?